crates/db/src/lib.rs

```diff
@@ -292,6 +294,24 @@ impl DB {
         Ok(res)
     }
 
+    pub async fn get_nft_tokens(&self, chain_id: u32) -> Result<Vec<NftToken>> {
+      let res: Vec<_> = sqlx::query(
+        r#" SELECT * 
+        FROM nft_tokens
+        WHERE chain_id = ?"#,
+      )
+      .bind(chain_id)
+      .map(|row| NftToken {
+        contract: Address::from_str(row.get::<&str, _>("contract")).unwrap(),
+        token_id: U256::from_dec_str(row.get::<&str, _>("token_id")).unwrap(),
+        owner: Address::from_str(row.get::<&str, _>("owner")).unwrap(),
+      })
```

Comment:
instead of converting here, this should be done by implementing `impl TryFrom<SqliteRow> for NftToken`. check `tokens.rs` for how it was done for the existing ones